packages/components/icons/src/leading-icon/leading-icon.styles.ts

```diff
@@ -8,7 +8,7 @@ type TColorThemeConfiguration = {
   border?: string;
 };
 
-const sizeMap = {
+export const sizeMap = {
```

Comment:
Do we actually need to export this constant?

Reply:
no, that must have slipped back in at some point, the export has been removed

Reply:
Thanks.